src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+    {person.isMarried ? (
+      <>
+        {person.sex === 'm' && (
+          <p className="Person__partner">{`${person.partnerName} is my wife`}</p>
+        )}
+        {person.sex === 'f' && (
+          <p className="Person__partner">{`${person.partnerName} is my husband`}</p>
```

Comment:
you repeat the same tag twice, use this ternary operator inside of this tag to not repeat yourself